pkg/controller/clusterinstallation/minio.go

```diff
@@ -28,6 +28,28 @@ func (r *ReconcileClusterInstallation) checkMinio(mattermost *mattermostv1alpha1
 }
 
 func (r *ReconcileClusterInstallation) checkMinioSecret(mattermost *mattermostv1alpha1.ClusterInstallation, reqLogger logr.Logger) error {
+	// Check if custom secret was specified
+	if mattermost.Spec.Minio.Secret != "" {
+		// Check if the Secret exists
+		var secret *corev1.Secret
+		err := r.client.Get(context.TODO(), types.NamespacedName{Name: mattermost.Spec.Minio.Secret, Namespace: mattermost.Namespace}, secret)
+		if err != nil {
+			// Secret do not exist
+			return err
```

Comment:
we could get a more helpful error with `errors.Wrap(err, "unable to locate custom minio secret")`

Reply:
Yes makes sense. I'll make the change.